examples/baselines/bc/bc_rgbd.py

```diff
@@ -0,0 +1,510 @@
+import datetime
+import os
+import random
+import uuid
+from dataclasses import asdict, dataclass
+from typing import Optional
+
+import gymnasium as gym
+import h5py
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import tyro
+import wandb
+from mani_skill.utils.io_utils import load_json
+from mani_skill.utils.wrappers.gymnasium import CPUGymWrapper
+from mani_skill.utils.wrappers.record import RecordEpisode
+from torch.utils.data import DataLoader, Dataset
+from mani_skill.vector.wrappers.gymnasium import ManiSkillVectorEnv
+from torch.utils.data.sampler import BatchSampler, RandomSampler
+from tqdm import tqdm
+
+
+@dataclass
+class Config:
+    # wandb project name
+    project: str = "maniskill"
+    # wandb group name
+    group: str = "maniskill"
+    # training dataset and evaluation environment
+    env: str = "PickCube-v1"
+    # demo path
+    demo_path: str = ""
+    # total gradient updates during training
+    max_timesteps: int = int(1e6)
+    # training batch size
+    batch_size: int = 128
+    # maximum possible trajectory length
+    max_traj_len: int = 100
+    # evaluation frequency, will evaluate eval_freq training steps
+    eval_freq: int = int(500)
+    # record videos
+    video: bool = False
+    # video save path
+    video_path = "./videos"
+    # how often a video should be recorded
+    video_freq: int = 1000
+    # training random seed
+    seed: int = 42
+    # training device
+    device: str = "cuda"
+    # learning rate
+    lr = 3e-4
+    # number of parallel eval envs
+    num_envs = 50
+    # dataset base directory
+    dir = "/root/"
+    # log to wandb
+    wandb: bool = True
+    # number of trajectories to load, max is usually 1000
+    load_count: int = 500
+    # seed
+    seed:int = 2024
+    # where experiment outputs should be stored
+    output_dir = "./output/"
+    # save frequency
+    save_freq: int = 10000
+    # simulation backend
+    sim_backend:str = "cpu"
+
+
+def load_h5_data(data):
+    out = dict()
+    for k in data.keys():
+        if isinstance(data[k], h5py.Dataset):
+            out[k] = data[k][:]
+        else:
+            out[k] = load_h5_data(data[k])
+    return out
+
+def make_mlp(in_channels, mlp_channels, act_builder=nn.ReLU, last_act=True):
+    c_in = in_channels
+    module_list = []
+    for idx, c_out in enumerate(mlp_channels):
+        module_list.append(nn.Linear(c_in, c_out))
+        if last_act or idx < len(mlp_channels) - 1:
+            module_list.append(act_builder())
+        c_in = c_out
+    return nn.Sequential(*module_list)
+
+def wandb_init(config: dict) -> None:
+    wandb.init(
+        config=config,
+        project=config["project"],
+        group=config["group"],
+        name=log_name.replace(os.path.sep, "__"),
+        id=str(uuid.uuid4()),
+        tags=["BC_RGBD"]
```

Comment:
for consistency lets name the tag "bc" for both bc rgbd and bc state, and also label group to be "BehaviorCloning". Default project name in the tyro args should be ManiSkill.

moreover why is there a ID generated here? We don't need to use uuid.